### ISSUE TITLE
NPE in QuarkusCxfProcessor#build (0.1.1)

### ISSUE THREAD

**Issue by jimbogithub** [quarkiverse/quarkiverse-cxf#75]
Following error occurs during build using 0.1.1.
```
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:1.10.3.Final:build (default) on project my-cxf-svc: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
[ERROR] 	[error]: Build step io.quarkiverse.cxf.deployment.QuarkusCxfProcessor#build threw an exception: java.lang.NullPointerException
[ERROR] 	at io.quarkiverse.cxf.deployment.QuarkusCxfProcessor.build(QuarkusCxfProcessor.java:1126)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[ERROR] 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:972)
[ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
[ERROR] 	at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:2046)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1578)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1452)
[ERROR] 	at java.base/java.lang.Thread.run(Thread.java:834)
[ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:479)
```

If you can provide any clues as to the potential issue I may be able to experiment / investigate further.

**Comment by dufoli** [comment]
very strange because error is on:

```
MethodParameterInfo paramInfo = methodAnnotation.target().asMethodParameter();
if (paramInfo != null && paramName.equals(paramInfo.name())) {

```
paramInfo is tested and paramName is the name of a method parameter which is obviously not null....
car you post the webmethod which lead to this issue ?
service interface if possible.

**Comment by Dufgui** [comment]
fix with 914e2f0b62bb3b1c94571c1f996c05a89919ab1a

**Comment by Dufgui** [comment]
please retry with the master

**Comment by jimbogithub** [comment]
> please retry with the master

Confirmed resolved in 0.3.0-SNAPSHOT.
